editor/editor_inspector.cpp

```diff
@@ -3329,7 +3329,11 @@ void EditorInspector::update_tree() {
 				if (use_doc_hints) {
 					// `|` separator used in `EditorHelpTooltip` for formatting.
 					if (theme_item_name.is_empty()) {
-						ep->set_tooltip_text("property|" + classname + "|" + property_prefix + p.name + "|");
+						if (p.usage & PROPERTY_USAGE_INTERNAL) {
+							ep->set_tooltip_text("internal_property|" + classname + "|" + property_prefix + p.name + "|");
```

Comment:
Should the tool tip read "internal property" instead of "internal_property" because it's a user-facing message?

Reply:
It's not a user facing message.

Reply:
In that case, totally fine :)
It seemed like it was setting a tool tip somewhere that the user would see.

Reply:
No, same as it is with the surrounding code, this text is parsed by the EditorHelpTooltip control and the actual content is different.